torchvision/datasets/utils.py

```diff
@@ -483,3 +484,39 @@ def verify_str_arg(
         raise ValueError(msg)
 
     return value
+
+
+def read_pfm(file_name: str, slice_channels: int = 2) -> np.ndarray:
+    """Read file in .pfm format. Might contain
+
+    Args:
+        file_name (str): Path to the file.
+        slice_channels (int): Number of channels to slice out of the file.
+        Useful for reading different data formats stored in .pfm files: Optical Flows, Stereo Disparity Maps, etc.
```

Comment:
Skipping a line in a parameter description requires to indent it to the right (otherwise sphinx doesn't render it properly):

```suggestion
            Useful for reading different data formats stored in .pfm files: Optical Flows, Stereo Disparity Maps, etc.
```